features/guidance/collapse.feature

```diff
@@ -425,18 +425,28 @@ Feature: Collapse
     Scenario: Pankenbruecke
         Given the node map
             """
-            j       h           i
-                        b c d e f           g
-            k       a
+            k j
+            | |
+            | |
+            | |
+            a h
+             b
+             c
+             d
+             e
+             f-i
+             |
+             |
+             |
+             g
             """
-
         And the ways
-            | nodes | highway | name    | oneway |
-            | kabhj | primary | inroad  | yes    |
-            | bc    | primary | inroad  | no     |
-            | cd    | primary | bridge  | no     |
-            | defg  | primary | outroad | no     |
-            | fi    | primary | cross   | no     |
+            | nodes | highway | name    | oneway | lanes |
+            | kabhj | primary | inroad  | yes    | 4     |
```

Comment:
If the lane tag is not present, the road will not be recognised as being the same. We will not merge the two into one. This changes turn angles / obvious detection. It can end up as a `slight turn` instead of straight, for example.